client/interface.go

```diff
@@ -26,9 +26,13 @@ type VerifyingLogClient interface {
 	// is available. If no proof is available within the ctx deadline, DeadlineExceeded
 	// is returned.
 	AddLeaf(ctx context.Context, data []byte) error
-	// GetByIndex returns a single leaf.
+	// VerifyInclusion ensures that data has been included in the log.
```

Comment:
Should we mention the verification mechanisms (ie, that it checks via inclusion proof), or are we assuming the client's audience understands that?